packages/plugin-pnp/sources/AbstractPnpInstaller.ts

```diff
@@ -1,9 +1,9 @@
-import {InstallStatus, Installer, LinkOptions, LinkType, MessageName, DependencyMeta, FinalizeInstallStatus} from '@yarnpkg/core';
-import {FetchResult, Descriptor, Locator, Package, BuildDirective}                                           from '@yarnpkg/core';
-import {miscUtils, structUtils}                                                                              from '@yarnpkg/core';
-import {FakeFS, PortablePath, ppath}                                                                         from '@yarnpkg/fslib';
-import {PackageRegistry, PnpSettings}                                                                        from '@yarnpkg/pnp';
-import mm                                                                                                    from 'micromatch';
+import {Installer, LinkOptions, LinkType, MessageName, DependencyMeta, FinalizeInstallStatus} from '@yarnpkg/core';
+import {FetchResult, Descriptor, Locator, Package, BuildDirective}                            from '@yarnpkg/core';
+import {miscUtils, structUtils}                                                               from '@yarnpkg/core';
+import {FakeFS, PortablePath, ppath}                                                          from '@yarnpkg/fslib';
+import {PackageRegistry, PnpSettings}                                                         from '@yarnpkg/pnp';
+import mm                                                                                     from 'micromatch';
```

Comment:
I know you really like this style of imports @arcanis, but it looks really  odd to me in smaller screens. That's unrelated to the PR, but just saying